grisha.pletin/SecondHW/valid?.rb

```diff
@@ -0,0 +1,14 @@
+class String
+  def valid?
+    stack = []
+    brackets = { '{' => '}', '[' => ']', '(' => ')', '<' => '>' }
+    each_char do |char|
+      stack << char if brackets.key?(char)
+      return false if brackets.key(char) && brackets.key(char) != stack.pop
+    end
+    stack.empty?
+  end
+end
+p '{(})'.valid?
+p '{{[]'.valid?
+p '{[()]}'.valid?
```

Comment:
Please add an empty line in the end of the file.
You need to do this because otherwise there is no way to tell the difference between a file where there is a newline at the end and one where is not. Diff has to output a newline anyway, or the result would be harder to read or process automatically.

Note that it is a good style to always put the newline as a last character if it is allowed by the file format. Furthermore, for example, for C and C++ header files it is required by the language standard.